server/app/migrations/versions/ba9a71425f36_.py

```diff
@@ -0,0 +1,68 @@
+"""empty message
+
+Revision ID: ba9a71425f36
+Revises: 138ae7ddca87
+Create Date: 2020-03-04 17:17:31.776338
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'ba9a71425f36'
+down_revision = '138ae7ddca87'
```

Comment:
Migration 285b17... also has the same down_revision. Maybe you should rebase and re-make this migration? Otherwise we'll have to look into how to merge them.